css/properties/-moz-image-region.json

```diff
@@ -0,0 +1,57 @@
+{
+  "css": {
+    "selectors": {
+      "-moz-image-region": {
+        "__compat": {
+          "description": "<code>:-moz-image-region</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/-moz-image-region",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": true,
+              "prefix": "-moz"
```

Comment:
Generally the prefix shouldn't be included when the feature name already includes it (e.g. #2832)